### PR TITLE
Use indent size of 4 for documentation

### PR DESCRIPTION
**Pull request by zanieb**
This is required for proper sub-bullet rendering in mkdocs

### CODE REVIEW

File: PIP_COMPATIBILITY.md [hunk 1]
```diff
@@ -364,8 +364,8 @@ does support a large subset.
 Missing options and subcommands are prioritized based on user demand and the complexity of the
 implementation, and tend to be tracked in individual issues. For example:
 
-- [`--trusted-host`](https://github.com/astral-sh/uv/issues/1339)
-- [`--user`](https://github.com/astral-sh/uv/issues/2077)
+-   [`--trusted-host`](https://github.com/astral-sh/uv/issues/1339)
+-   [`--user`](https://github.com/astral-sh/uv/issues/2077)
```
Comment (zanieb) [inline PIP_COMPATIBILITY.md:368]: I cannot comprehend why this is changed.

Comment (zanieb) [reply PIP_COMPATIBILITY.md:368]: It renders fine though.

Comment (zanieb) [reply PIP_COMPATIBILITY.md:368]: Tragic:

- https://github.com/prettier/prettier/pull/15526
- https://github.com/prettier/prettier/issues/5019

**Comment by zanieb** [comment]
It's horrible, but I don't know what else to do if we want to use prettier.

**Comment by zanieb** [comment]
Going to use `<!-- prettier-ignore -->` for nested lists for now instead. Please watch out for this footgun.

**Comment by glenn-jocher** [comment]
I'm waiting for https://github.com/ultralytics/ultralytics/pull/15619 to merge, but it's taking ages unfortunately.

**Pull request closed without merging**